addons/point_of_sale/static/src/js/models.js

```diff
@@ -522,7 +522,7 @@ class PosGlobalState extends PosModel {
     }
 
     send_current_order_to_customer_facing_display() {
-        if (!this.config.iface_customer_facing_display) return;
```

Comment:
Are you sure we don't want to send to customer display when no order? Because when you are finalizing an order and then you don't have one anymore, we should "clear" the screen. (or maybe we always have an empty order, so in which case don't we have an order?

Reply:
True true.